courseScheduleII.js

```diff
@@ -0,0 +1,110 @@
+/*
+For this approach, we can represent the input as a graph where in order to take a course, 
+you have to complete a prereq, which is the same as two nodes conntected by an edge.
```

Comment:
This is called a "dependency graph". It's probably sufficient to say "We can represent the input as a dependency graph".

Note: dependency graphs are always directed, so there's no need to say that it's directed.